### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project,

- [ ] I created an npm project with webpack.
- [ ] I wrote plain HTML markup with minimum styling (just to make the layout).
- [ ] I used ES6 modules, with import and export.
- [ ] I optionally added running time using luxon
- [ ] I optionally added a remove button for the scores but hidden with CSS

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      <input type="text" id="name" placeholder="Your name" />
+      <label for="score" hidden>Score:</label>
+      <input type="text" id="score" placeholder="Your score" />
+      <button type="submit" id="add-btn" class="add-btn">Submit</button>
+    </form>
+  </div>
+  <span id="message"></span>
+</body>
+<!-- <script src="./index.js" type="module"></script> -->
+</html>
```
Comment (muneebulrehman) [inline src/index.html]: As per the project requirements, your design should match the wireframe design provided. You don't need to go crazy with the styling, but the bare minimum should be matched with the wireframe design.

| Your design  | Wireframe design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @anyars-encarta , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Project setup with webpack. 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (anyars-encarta) [reply src/index.html]: Thank you very much for the CR. I will definitely do that

**Review by V-Blaze — APPROVED** [review]
Hi @anyars-encarta  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Webpack setup correctly💯
-  Good communication with reviewer 👍👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**